### ISSUE TITLE
Contributor needed

### ISSUE THREAD

**Issue by wcandillon** [wcandillon/swagger-js-codegen#167]
Unfortunately, I am not using this project anymore and I'm having trouble keeping up with all its pull requests, would someone be interested in becoming a contributor to this project?

**Comment by danmarshall** [comment]
Perhaps the [TypeStrong](https://github.com/TypeStrong) org?

**Comment by kibertoad** [comment]
@wcandillon I would be interested in joining as a contributor. What is your estimate for amount of time needed weekly to be of use? I think I could be doing something similar to 10 hours a week, more when working on some task which is related to my day work.

**Comment by wcandillon** [comment]
@kibertoad Any amount of help would be appreciated 🙋🏼‍♂️

**Comment by restrry** [comment]
@wcandillon does ie mare sense to move the library into an organisation and add some volunteers? I forked another fork as I need only ts generator, but would like to contribute here as well

**Comment by kastentx** [comment]
@wcandillon I have a current interest in this project and would be interested in taking over maintenance. Are you still looking to hand this off?

**Comment by wcandillon** [comment]
yes I'm very much interested in that. Do you know if there would be a way for you as well to publish new version of the package on npm?
